### ISSUE TITLE
Cannot analyze external checker report

### ISSUE THREAD

**Issue by HassanQayyum1993** [SonarOpenCommunity/sonar-cxx#1773]
### Description

After creating report from an external checker and tranmsitting to Relax NG schema I cannot analyze this report and its issues, which are active in my quality profile are not visible in sonarqube report. 

### Steps to reproduce the problem
1. Created xml report with following syntax
```xml
<?xml version="1.0" encoding="ASCII"?>
 <element name="results" xmlns="http://relaxng.org/ns/structure/1.0">
   <zeroOrMore>
     <element name="error">
       <attribute name="myfile.cpp"/>
       <attribute name="msg"/>
      <attribute name="id"/>
       <attribute name="30">
         <data type="integer" datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes" />
       </attribute>
       <text/>
     </element>
  </zeroOrMore>
 </element>
```
2. Created rule in Sonarqube external rule section with following syntax (rule created successfully).
```xml
<?xml version="1.0" encoding="ASCII"?>
 <rules>
 <rule>
 <key>id</key>
   <name>msg</name>
    <description>msg</description>
   </rule>
 </rules>
```
### Expected behavior
Sonar Qube should report the error in its report after running analyses.

### Actual behavior
Look like the sonar scanner not able to read the report. following is what appears in cmd while running scanner.

###Log
09:22:06.046 INFO: Sensor C++ (Community) ExternalRulesSensor [cxx]
09:22:06.046 INFO: Searching reports by relative path with basedir 'C:\ContinuousIntegration\Repositories\CT_Driver\Trunk\Library' and search prop 'sonar.cxx.other.reportPath'
09:22:06.046 DEBUG: Normalized report includes to '[C:\DevSkimreport.xml]'
09:22:06.046 DEBUG: Scanner uses normalized report path(s): 'C:\DevSkimreport.xml'
09:22:06.046 INFO: Parser will parse '1' report file(s)
09:22:06.046 INFO: Processing report 'C:\DevSkimreport.xml'
09:22:06.046 DEBUG: Parsing 'other' format
09:22:06.046 DEBUG: CXX-OTHER processed = 0
09:22:06.046 INFO: CXX-OTHER processed = 0
09:22:06.046 INFO: Sensor C++ (Community) ExternalRulesSensor [cxx] (done) | time=0ms
09:22:06.046 INFO: Sensor JaCoCo XML Report Importer [jacoco]
09:22:06.046 DEBUG: No reports found
09:22:06.046 INFO: Sensor JaCoCo XML Report Importer [jacoco] (done) | time=0ms
09:22:06.046 INFO: Sensor JavaXmlSensor [java]

* cxx plugin version 1.3.1.1807
* SonarQube version 7.9.1

**Comment by guwirth** [comment]
Hi @HassanQayyum1993,

please have a look to our rdwiki. Think it's wrong XML format.
https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Extending-the-code-analysis#usage-of-unsupported-code-checkers

Regards,

**Comment by jehanzebqayyum** [comment]
`Hi guwirth, 
I have followed the same format.

**Comment by HassanQayyum1993** [comment]
Hi guwirth,
I have used the same format.

**Comment by guwirth** [comment]
Hi @HassanQayyum1993,

can you please verify:
1) Is the rule you wanna check visible in the SQ UI Rules section?
2) Did you 'Activate' the rule/rules in a Quality Profile?
3) Do you use the right Quality Profile for the analysis?
4) Is rule id the same / in the report?
5) Turn debug information on: https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Get-debug-information

Regards,
